<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>Computational Biology Seminar Series</title>
		<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css">
		<link href='http://fonts.googleapis.com/css?family=Roboto:400,300,500,700' rel='stylesheet' type='text/css'>
		<link rel="stylesheet" href="css/style.css">
		<script>
		  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		  ga('create', 'UA-43635419-1', 'lsu.edu');
		  ga('send', 'pageview');

		</script>
	</head>
	<body class="container">
		<header>
			<img src="img/banner.jpg" class="img-rounded banner">
			<h2><strong>Computational Biology Seminar Series for Undergraduates</strong></h2>
			<p>
				<em>Sponsored by the <a href="http://science.lsu.edu/">LSU College of Science</a>, the <a href="http://www.biology.lsu.edu">Department of Biological
Sciences</a>, the <a href="http://cct.lsu.edu/">Center for Computation & Technology</a> and the <a href="http://lbrn.lsu.edu">Louisiana Biomedical Research Network</a></em>
			</p>
		</header>
				<div class="row">
			<div class="col-lg-8 col-md-8">
				<h3>Divide and Conquer: one computational approach to solving the greatest challenge in evolutionary biology</h3>
<!--				<p style="text-align: center">
					<iframe width="420" height="315" src="//www.youtube.com/embed/NY_K7lDXiM4" frameborder="0" allowfullscreen></iframe>
				</p>
-->

				<h4>Abstract:</h4>
				<p>Reconstructing and understanding the relationships that unite all life has been a goal of biologists for many years. Computational limits and the lack of homologous data across distantly related lineages makes direct inference unrealistic. In the face of this difficulty, researchers have compiled tens of thousands of evolutionary trees that describe subsets of the tree of life. Until recently, no one has taken on the task of developing methods to combine this data into a more complete phylogeny. The research team for the Open Tree of Life has built the first openly available phylogeny composed of data from a number of taxonomic resources and evolutionary trees into a draft tree of life containing over 2 million species.</p>
<h4>Bio:</h4>
<p> Lyndon M Coghill, PhD., is currently a postdoctoral researcher in the department of Biological Sciences at Louisiana State University, where his research focuses on understanding models of molecular evolution and methods of phylogenetic inference. Outside of this work, Lyndon’s research focuses on a broad range of topics, including the evolutionary history of trait adaptation for extreme environments, patterns of coevolution between species, and historic patterns of population structure. In general, Lyndon’s research interests are centered around understanding and identifying biodiversity, what forces drive patterns of evolutionary history, and what are the best methods to discover those patterns. Lyndon holds an undergraduate degree from Western Illinois University, a PhD from the University of New Orleans and prior to joining LSU, was a postdoctoral fellow at the Field Museum of Natural History in Chicago. </p>
				
			</div>
			<div class="col-lg-4 col-md-4">
				<img src="img/coghill.jpg" class="pull-right">
				<h3><a href="http://www.phyleaux1.lsu.edu/?q=node/64">Lyndon M Coghill</a></h3>
				<p>Postdoctoral Researcher, <a href="http://www.phyleaux1.lsu.edu">Phyleaux Lab</a>, Louisiana State University</p>
				

			</div>
		</div>
	
		<ul class="pager">
			<li class="previous"><a href="http://lbrn.lsu.edu/events/comp-bio">&larr; Back to All Seminars</a></li>
		</ul>
		<script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
		<script src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
	</body>
</html>
