<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>Computational Biology Seminar Series</title>
		<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css">
		<link href='http://fonts.googleapis.com/css?family=Roboto:400,300,500,700' rel='stylesheet' type='text/css'>
		<link rel="stylesheet" href="css/style.css">
		<script>
		  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		  ga('create', 'UA-43635419-1', 'lsu.edu');
		  ga('send', 'pageview');

		</script>
	</head>
	<body class="container">
		<header>
			<img src="img/banner.jpg" class="img-rounded banner">
			<h2><strong>Computational Biology Seminar Series for Undergraduates</strong></h2>
			<p>
				<em>Sponsored by the Department of Biological Sciences and the Center for Computation & Technology</em>
			</p>
		</header>
		<div class="row">
			<div class="col-lg-8 col-md-8">
				<h3>From shotguns to shotgun sequencing: Natural history collections provide “big data” for comparative genomics</h3>
				
				<h4>Abstract:</h4>
				<p>
					Natural history museums provide a wealth of resources for biological investigation, and new DNA sequencing technologies allow us to use genomic methods to unlock even more of their secrets. Museum collections are useful because they contain many individuals from thousands of species, which allows researchers to address questions using a comparative framework. The application of genomic methods to comparative studies, however, produces huge datasets that present bioinformatics and computational challenges. I will discuss how I, as a graduate student studying birds at the LSU Museum of Natural Science, combine 19th Century methods for obtaining museum samples with 21st Century laboratory and bioinformatics techniques. I will explain some of the computational challenges we have encountered and methods we are developing to overcome them. I will discuss how genomic datasets from museum collections are providing insights never before possible, and review the diverse opportunities that exist for the application of computational approaches to the treasure trove of data in natural history museums.
				</p>
			</div>
			<div class="col-lg-4 col-md-4">
				<img src="img/harvey.jpg" class="bio-picture pull-right">
				<h3>Mike Harvey</h3>
				<p><a href="http://www.mharvey.org/">LSU Department of Biological Sciences</a></p>
				<p>PH.D. Student</p>
				<script>
					var x     = new Date();
					var today = new Date();
					x.setFullYear(2014,1,20);

					if (today.getTime() < x.getTime()) {
					 	document.write('<div class="well"><h4>Live Feed:</h4><a href="https://seevogh.com/?join=1193648733" class="btn btn-primary"><img style="width: 20px" src="img/seevogh.png"> SeeVogh (<strong>HD</strong>)</a> </p> <p> No Access Code/No Login required, online link does not begin until seminar does. If this is your first SeeVogh meeting follow this link for a brief tutorial: <a href="https://seevogh.com/how-it-works/tutorials/">https://seevogh.com/how-it-works/tutorials/</a> </p> <p> System Requirements: PCs running either Windows, Mac or Linux operating systems. Note: if you’re using a Mac, due to Java permissions - you may want to use the “Start (Mac Beta)” app provided by SeeVogh here: <a href="http://research.seevogh.com/">http://research.seevogh.com/</a> </p> <p> <em>Note: if you plan to use an iOS or Android device, download the appropriate app via the http://seevogh.com/ website and then join the meeting # <strong>1193648733</strong> </em></p></div>');
					}
				</script>
			</div>
		</div>
		<ul class="pager">
			<li class="previous"><a href="http://lbrn.lsu.edu/events/comp-bio">&larr; Back to All Seminars</a></li>
		</ul>
		<script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
		<script src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
	</body>
</html>
