<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>Computational Biology Seminar Series</title>
		<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css">
		<link href='http://fonts.googleapis.com/css?family=Roboto:400,300,500,700' rel='stylesheet' type='text/css'>
		<link rel="stylesheet" href="css/style.css">
		<script>
		  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		  ga('create', 'UA-43635419-1', 'lsu.edu');
		  ga('send', 'pageview');

		</script>
	</head>
	<body class="container">
		<header>
			<img src="img/banner.jpg" class="img-rounded banner">
			<h2><strong>Computational Biology Seminar Series for Undergraduates</strong></h2>
			<p>
				<em>Sponsored by the <a href="http://science.lsu.edu/">LSU College of Science</a>, the <a href="http://www.biology.lsu.edu">Department of Biological
Sciences</a>, the <a href="http://cct.lsu.edu/">Center for Computation & Technology</a> and the <a href="http://lbrn.lsu.edu">Louisiana Biomedical Research Network</a></em>
			</p>
		</header>
				<div class="row">
			<div class="col-lg-8 col-md-8">
				<h3>Snake Sisters: how I use genes, models, and morphology to understand what we know about squamate evolution</h3>
<!--				<p style="text-align: center">
					<iframe width="420" height="315" src="//www.youtube.com/embed/NY_K7lDXiM4" frameborder="0" allowfullscreen></iframe>
				</p>
-->
				<h4>Abstract:</h4>
				<p>Herpetologists agree that squamates are a group encompassing all extant snake and lizard families. However, there is disagreement over how these families evolved and how they are related to each other. There have been over half a dozen papers published in the last decade with different types of genetic and morphological data and analyses, yet there is no consensus. This is the type of question that got me interested in the computational side of biology. As a lab technician working in conservation biology I learned how many different answers can come out of one dataset when different analyses were applied. I started to question how the collection, treatment, and analysis of genetic data affected the conclusions we were drawing. How do different types of genetic loci fit differently to the evolutionary models? Do some loci have so much information content that they overwhelm the rest of the data? How do we even begin to answer these questions? I am currently trying to answer these questions in terms of relationships between lizard and snake families. I will go over my current research and how I use computers to answer questions I developed as a conservation biologist. </p>


				
				<h4>Bio:</h4>
				<p>Genevieve is currently a Ph.D. student at Louisiana State University working with Dr. Jeremy Brown and Dr. Chris Austin. She has a B.S. in Evolution, Ecology, and Biodiversity from University of California, Davis. After graduating she started working as a laboratory technician and manager in the Shaffer Lab at the University of California Los Angeles. Her work focused on developing and troubleshooting novel methods of next-generation data collection in herpetological non-model organisms. The trials and tribulations of sample and data collection in non-model systems led her to pursue a PhD at LSU working with phylogenetic and species delimitation methods in herpetological systems. She is interested in better understanding how to analyze and understand variation within datasets. In particular she is interested in the current wealth of genome-wide data and parsing out biologically informative signal from systematic error. </p>
				
				
				
			</div>
			<div class="col-lg-4 col-md-4">
				<img src="img/mount.jpg" class="pull-right">
				<h3><a href="http://www.phyleaux1.lsu.edu/?q=node/55">Genevieve Mount</a></h3>
				<p>Louisiana State University, Computational Phylogenetics, Phylogeography and Molecular Evolution</p>
			
			<script>
				var x = new Date();
                var today = new Date();
               x.setFullYear(2016,10,10);
                
                if (x.toString() == today.toString())  {
					 	document.write('<div class="well"><h4>Live Feed:</h4><a href="https://viewme.ezuce.com/?join=2815191916" class="btn btn-primary"><img style="width: 20px" src="img/viewme.png"> Viewme (<strong>HD</strong>)</a> </p> <p> No Access Code/No Login required, online link does not begin until 30 minutes before seminar does. If this is your first Viewme meeting follow this link for a brief tutorial: <a href="https://viewme.ezuce.com/support/guides-tutorials/">https://viewme.ezuce.com/support/guides-tutorials/</a> </p> <p> System Requirements: PCs running either Windows, Mac or Linux operating systems. </p> <p> <em>Note: if you plan to use an iOS or Android device, download the appropriate app via the <a href="https://viewme.ezuce.com/download/">https://viewme.ezuce.com/download/</a> website and then join the meeting # <strong>2815191916</strong> </em></p></div>');
					}
					</script>
			
		</div>
	
		<ul class="pager">
			<li class="previous"><a href="http://lbrn.lsu.edu/events/comp-bio">&larr; Back to All Seminars</a></li>
		</ul>
		<script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
		<script src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
	</body>
</html>
