<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8">
		<title>Computational Biology Seminar Series</title>
		<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css">
		<link href='http://fonts.googleapis.com/css?family=Roboto:400,300,500,700' rel='stylesheet' type='text/css'>
		<link rel="stylesheet" href="css/style.css">
		<script>
		  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		  ga('create', 'UA-43635419-1', 'lsu.edu');
		  ga('send', 'pageview');

		</script>
	</head>
	<body class="container">
		<header>
			<img src="img/banner.jpg" class="img-rounded banner">
			<h2><strong>Computational Biology Seminar Series for Undergraduates</strong></h2>
			<p>
				<em>Sponsored by the Department of Biological Sciences and the Center for Computation & Technology</em>
			</p>
		</header>
		<div class="row">
			<div class="col-lg-8 col-md-8">
				<h3>Weâ€™re not in a petri-dish anymore: pulling back the curtain on microbial genomics using cultivation-independent techniques</h3>
				<script>
					var x = new Date();
					var today = new Date();
					x.setFullYear(2013,11,2);

					if (x.toString() == today.toString()) {
					 	document.write('<p style="text-align: center; margin-bottom: 20px"> <h4>Live Feed:</h4> <a href="http://connect.lsu.edu/compbio" class="btn btn-info"><img style="width: 20px" src="img/adobe.png"> Adobe Connect</a> OR <a href="https://seevogh.com/?join=9492783223" class="btn btn-primary"><img style="width: 20px" src="img/seevogh.png"> SeeVogh (<strong>HD</strong>)</a> </p>');
					}
				</script>
				<h4>Abstract:</h4>
				<p>
					The study of environmental microorganisms is critical to understanding global biogeochemical cycles. However, microorganisms present unique challenges for linking individuals to environmental function, and isolating important members of microbial communities in culture is unreliable. Whole-community DNA and RNA sequencing (metagenomics and metatranscriptomics) has changed the way in which modern microbiologists approach studying microorganisms in their native environments. In combination with new single-cell genomics techniques and large computational infrastructure, these techniques providing unprecedented access to previously unknown functional capabilities of microorganisms in nature. This talk will describe the progression in cultivation-independent techniques for studying microbes from single-gene surveys to reconstruction of entire genomes from community DNA and even from individual cells. We will explore the benefits of these new techniques with specific examples of how they have shed light on previously unknown microbial taxa and processes.
				</p>
			</div>
			<div class="col-lg-4 col-md-4">
				<img src="img/thrash.jpg" class="pull-right" style="width: 170px">
				<h3>Cameron Thrash</h3>
				<p>LSU Department of Biological Sciences</p>
				<p>Assistant Professor</p>
				<p><a href="http://thethrashlab.com/">http://thethrashlab.com/</a></p>
			</div>
		</div>
		<ul class="pager">
			<li class="previous"><a href="http://lbrn.lsu.edu/events/comp-bio">&larr; Back to All Seminars</a></li>
		</ul>
		<script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
		<script src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
	</body>
</html>
